src/pages/player-info.md

```diff
@@ -0,0 +1,98 @@
+
+The game will be streamed to [Matty’s Twitch channel](https://www.twitch.tv/mattstratton/), which is how the “audience” can watch. The Twitch channel will display the main screen of the game, plus indicators of who is in the Discord room and when they are speaking. All audio from the game (plus the Discord call) will also be streamed to Twitch, so that contestants will be able to be heard.
+
+Please note: your camera **will** be on for the stream, so be “camera ready” (whatever that means to you!). If you would prefer to NOT have your camera show in the stream, please let Matty know.
+
+People watching the game can join the “audience” of the game on their personal devices; this lets them vote/participate in the rankings for the game as well! The audience doesn’t “play the game” but they vote on the players’ responses and can potentially influence the winners!
+
+There will be 6-8 players, who will play both rounds. The hosts (Matty, Jeremy, Dan) will not be playing (it would be slightly unfair since we wrote the prompts!)
+
+## Discord
+If you are not already in our Discord server, please join with the invite link at [https://devopspartygames.com/discord](https://devopspartygames.com/discord). You will need to accept the Code of Conduct to get access to the server. **Please let an admin know when you have joined, and we will grant you the “Players” role that gives you more access in Discord.** We will be capturing/streaming the audio from the Discord chat to the livestream. Also, bear in mind that your Discord username and avatar will be displayed on the stream (it will “light up” when you are speaking; see the screenshot below for an example)
+
+![](/images/discord-overlay-example.jpg)
+
+Once you join Discord, send a note anyone listed under Admins in the Discord (Matty, Jeremy, or Dan) and let us know you are in; we will grant you the proper role to give you access to the “Game” voice channel (and the “players” text channel). The “Game” voice channel is what you will need to join/connect audio to during the actual stream/event. The text channel is just in case we want it. 
+
+You also should set your “nickname” on our Discord server to your actual name - that is what will display in the overlay instead of your Discord username. It will just look nicer :)
+
+Note that the Discord is public for members of the community; there are private channels just for players and moderators though!
+
+## Schedule
+Episodes are streamed at **20:00 UTC+1** (EMEA league) or **20:00 UTC-6** (Americas league) on a **Tuesday**!  The game itself (all rounds) will likely take about an hour, but we will schedule an extra 30 minutes prior for “green room” time to make sure everything is working as expected. Please join the Discord voice channel (“Game”) **30 minutes before show time on game night** so we can do tech checks, etc.
```

Comment:
Those UTC markers are valid for standard time only; summer time shifts them both by +1.

```suggestion
Episodes are streamed at **20:00 CET / Paris** (EMEA league) or **20:00 CT / Chicago** (Americas league) on a **Tuesday**!  The game itself (all rounds) will likely take about an hour, but we will schedule an extra 30 minutes prior for “green room” time to make sure everything is working as expected. Please join the Discord voice channel (“Game”) **30 minutes before show time on game night** so we can do tech checks, etc.
```